build-system/tasks/presubmit-checks.js

```diff
@@ -596,6 +596,10 @@ const forbiddenTerms = {
       'extensions/amp-access/0.1/iframe-api/access-controller.js',
     ],
   },
+  '(?<!return) allowConsoleError\\(\\(\\) => \\{[\\n\\r\\s]*return': {
```

Comment:
`\\(\\)=> \\{[\\n\\r\\s]*return` fails to capture expression bodies, non-arrows, parameters, any anything that's not immediately returning.

Reply:
That's true. It's a simple check for the kind of usage we currently have. What do you suggest this be changed to?

Reply:
An eslint plugin.

Reply:
Okay, that's out of scope of this PR, so I'll revert this presubmit check and *cough* let you write a new plugin since you're more familiar with this than I am :)